contracts/protocol/tokenization/StaticATokenLM.sol

```diff
@@ -379,14 +378,15 @@ contract StaticATokenLM is
       return;
     }
     uint256 rewardsIndex = _getCurrentRewardsIndex();
+    require(from != to, 'Sender and recipient addresses should be different.');
```

Comment:
The concern of Ernesto was about not updating the state 2 times, not about reverting tx, will fix it after the merge